scripts/build

```diff
@@ -24,6 +24,7 @@ if [ "$CROSS" = 1 ]; then
     echo Built ./bin/dapper-$(uname -s)-$(uname -m)
 elif [ "$1" = "install" ]; then
     CGO_ENABLED=0 go install -a -tags netgo -installsuffix netgo -ldflags "-X main.VERSION=$VERSION -extldflags '-static -s'" ./...
+    mv "$GOPATH"/bin/rancher-dapper "$GOPATH"/bin/dapper
```

Comment:
is this needed. go install will do this for you

Reply:
go-install does not have an option to override the output filename (like go-build does).  By default go-install outputs the filename based on the module name.